### ISSUE TITLE
ip-api detection broken

### ISSUE THREAD

**Issue by FarSeeing** [AliasIO/wappalyzer#4710]
Current detection relies on `xhr` only:
```
  "ip-api": {
    "cats": [
      79
    ],
    "icon": "ip-api.png",
    "pricing": [
      "freemium",
      "low",
      "payg"
    ],
    "saas": true,
    "website": "https://ip-api.com/",
    "xhr": "ip-api\\.com/"
  },
```
but it doesn't work because of the trailing slash - the hostname coming from XHR request doesn't contain it.

**Comment by nurbek91** [comment]
Hey there , can you share on which sites this technology is not correctly identified.

**Comment by AliasIO** [comment]
Fixed in aa75a43286fd83275de3a2722139f1aa9f915c32, thanks!
